ord_schema/visualization/drawing.py

```diff
@@ -80,7 +80,10 @@ def mol_to_png(mol, max_size=1000, bond_length=25, png_quality=70):
     molecule."""
```

Comment:
update the comment too?

Reply:
Updating the docstrings for all visualization modules (in another PR).